src/main/java/com/kims/community/config/JwtAuthenticationProvider.java

```diff
@@ -0,0 +1,93 @@
+package com.kims.community.config;
+
+import com.kims.community.users.entity.Users;
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jws;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+import java.time.Duration;
+import java.util.Base64;
+import java.util.Date;
+import javax.crypto.SecretKey;
+import javax.crypto.spec.SecretKeySpec;
+import javax.servlet.http.HttpServletRequest;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class JwtAuthenticationProvider {
+
+    @Value("${jwt.secretkey}")
+    private String SECRETKEY;
+
+    public String generateToken(Users users, Duration expiredAt) {
```

Comment:
```suggestion
    public String generateToken(Users users, Duration expirationPeriod) {
```
expiredAt 보다는 유효기간(시간범위) 를 나타내는 expirationPeriod 혹은 expirationDuration 과 같은 네이밍은 어떠신가요?

실제로 만료날짜(expiredAt) 을 나타내는 변수가 아니라 만료기간을 나타내는 기간범위에 대한 변수명으로 보이기 때문입니다.

동건님이 리뷰주신 코멘트대로 현재시간은 createToken 내부에서 해도 좋아보입니다. 결과적으로 현재시간을 구하는 이유가 만료날짜를 구하기 위함이니까요 : )